clients/ts/FunctionalTests/ts/TestLogger.ts

```diff
@@ -34,7 +34,7 @@ export class TestLogger implements ILogger {
     public currentLog: TestLog = new TestLog();
 
     public log(logLevel: LogLevel, message: string): void {
-        this.currentLog.addMessage(logLevel, message);
+        this.currentLog.addMessage(new Date(), logLevel, message);
```

Comment:
Is this local time or UTC? I'd prefer UTC

Reply:
Sure, I can do that. We only really care about the elapsed time.

Reply:
`new Date` is UTC, turns out :).